### ISSUE TITLE
Seeking Guidance: Replication Challenges and Environment Clarification for Your Research

### ISSUE THREAD

**Issue by Iris-Zhang918** [zylbuaa/TFormer#1]
Hello, I am very interested in your research and attempting to replicate your code. I have run it several times with all the default parameters provided in the current repository, but the average accuracy is around 76.3%. I would like to ask if you have any suggestions on what should I do to get the same results as mentioned in the paper. Additionally, could you provide more detailed environment files to help rule out any environmental issues? Thank you very much.

**Comment by zylbuaa** [comment]
Thank you for expressing interest in our work. I have updated the requirements.txt file, which contains detailed information about the required environments. Due to the small size of the Derm7pt dataset, the model is prone to overfitting, potentially leading to unsatisfactory results. To address this issue, I recommend saving multiple models during one training process, such as the top 5 performing models on the validation set, and conducting multiple tests.

Since I have already graduated, the previously trained model has been deleted. However, I have retrained a new model, and its performance is comparable to that described in the article. You can download the model (new_model.pth) from the following link: "https://huggingface.co/vemvet/TFormer/tree/main" and use the test.py script for testing.

**Comment by maomao628** [comment]
> Hello, I am very interested in your research and attempting to replicate your code. I have run it several times with all the default parameters provided in the current repository, but the average accuracy is around 76.3%. I would like to ask if you have any suggestions on what should I do to get the same results as mentioned in the paper. Additionally, could you provide more detailed environment files to help rule out any environmental issues? Thank you very much.

hello，Could you please tell me the process of reproducing the code? I have some questions that I would like to ask you.This is my email:19860912916@163.com   Thank you very much.
